doc/adr/0011-add-google-sitelinks-searchbox-metadata.md

```diff
@@ -0,0 +1,20 @@
+# 11. Add Google Sitelinks Searchbox metadata
+
+Date: 2018-01-23
+
+## Status
+
+Accepted
+
+## Context
+
+Metadata can be added to a web site's home page to enable a search box directly within Google.
+
+## Decision
+
+Adding the required metadata is straight forward as the sites' search URL is the correct format for the Sitelinks Searchbox.
+An additional term will be added to the query string to allow analytic metrics to distinguish Google searches from searches initiated from the main site.
+
+## Consequences
+
+Additional searches will come from Google rather than the Pharmacy Finder homepage.
```

Comment:
This is being picky I know but I'd have the consequence something like this `Some searches will originate from Google, reducing the number of searches originating from the Pharmacy Finders' search page.`.